behat.yml.dist

```diff
@@ -72,17 +72,14 @@ default:
         header with menu: ".ecl-site-header-core[data-ecl-has-menu], .ecl-site-header-standardised[data-ecl-has-menu]"
         language dialog: ".ecl-dialog"
         language switcher: ".ecl-language-list"
-        language page switcher: ".ecl-page-header-core .ecl-lang-select-page"
         navigation: ".ecl-menu"
         page: "body"
-        page header: ".ecl-page-header-core, .ecl-page-header-standardised"
+        page header: ".ecl-page-header"
```

Comment:
Same as above.

Reply:
"page header" region is used to take the header. ".ecl-page-header-core, .ecl-page-header-standardised" means ".ecl-page-header-core" or ".ecl-page-header-standardised". Since we have the same class for both headers we don't need to separate them anymore.